### PR TITLE
Add types for create-cert

### PR DESCRIPTION
**Pull request by midgleyc**
Add types for `create-cert`. Limited to TypeScript 3.4 and above, due to an error when running dtslint:

```
Error: Errors in typescript@3.3 for external dependencies:
 ../node/globals.global.d.ts(1,44): error TS2304: Cannot find name 'globalThis'.
```

I found this odd because the docs say that only 3.6+ are tested in the first place.

Uses the types for `pem`.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Adding a new package
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/create-cert/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Type definitions for create-cert 1.0
+// Project: https://github.com/lukechilds/create-cert
+// Definitions by: Chris Midgley <https://github.com/midgleyc>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.4
```
Comment (peterblazejewicz) [inline types/create-cert/index.d.ts:5]:
```suggestion
```
I'd argue you no longer need this. It's not used during the CI already
and you're not referencing anything directly

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@midgleyc thx!

**Comment by midgleyc** [comment]
Ready to merge

File: types/create-cert/index.d.ts (hunk 1 above)
Comment (midgleyc) [reply types/create-cert/index.d.ts:5]: Sorry, missed this comment, only saw the tick below.

When I ran `npm test create-cert` from the root of the directory, it failed with this error -- I had to add a minimum TypeScript version to get it to run at all, and then it raised some errors for me to fix. Does the CI run something else?

Comment (peterblazejewicz) [reply types/create-cert/index.d.ts:5]: runs same, but maybe you're local TS repo was outdated. I've run PR locally, no problems

**Pull request merged**